src/core/migrations/0083_update_revision_request_template.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.19 on 2023-08-31 20:17
+
+from django.db import migrations
+from django.conf import settings
+from django.utils import translation
+
+SETTING_TEXTS_TO_REMOVE = [
+    ' Details of the changes required are below:<br><br>{{ revision.editor_note }}',
+    ' Details of the changes required are below:<br/><br/>{{ revision.editor_note }}'
+]
```

Comment:
Do we really want to remove this? It is a change on behavour that some editors may not like/expect. It would cause less friction to tweak the template in the cases where editors prefer it not be shared in the outgoing email notification

Reply:
Personally I think removing it makes sense. No other email templates work in this manner so its a bit of an odd one out actually.